src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1166,8 +1166,6 @@ private void ParseModifiers(SyntaxListBuilder tokens, bool forAccessors, bool fo
                         {
                             // Standard legal cases.
                             modTok = ConvertToKeyword(this.EatToken());
-                            modTok = CheckFeatureAvailability(modTok,
-                                isPartialType ? MessageID.IDS_FeaturePartialTypes : MessageID.IDS_FeaturePartialMethod);
```

Comment:
Do you want to adjust or remove the comments on the `MessageID` switch (in `RequiredVersion`) now or in a later PR?

Reply:
i can do here :)

Reply:
done.

Reply:
Sorry, I left that comment on a line as illustration, but it applies to all affected MessageIDs in this PR. Might be a good time to just remove the comments in `RequiredVersion`, rather than manually tracking them done one by one.